desktop/ui/src/main/java/org/datacleaner/windows/AnalysisJobBuilderWindowImpl.java

```diff
@@ -255,6 +260,8 @@ public void onRemove(final InputColumn<?> sourceColumn) {
 
     private static final Logger logger = LoggerFactory.getLogger(AnalysisJobBuilderWindow.class);
     private static final ImageManager imageManager = ImageManager.get();
+    
+    public static final List<Func<AnalysisJobBuilderWindowImpl, JComponent>> PLUGGABLE_LABELS_BANNER_COMPONENTS = new ArrayList<>(0);
```

Comment:
I could use the ServiceLoader, but wouldn't that be a mix of a class in the enterprise edition with the community edition ?